app/javascript/data/advancedFilterOptions.js

```diff
@@ -289,5 +289,37 @@ export const advancedFilterOptions = [
         type: 'date'
       }
     ]
+  },
+  {
+    name: 'vaccination',
+    title: 'Vaccination (Multi-select)',
+    description: 'Monitorees with specified Vaccination criteria',
+    type: 'multi',
+    tooltip: 'Returns records that contain at least one Vaccination entry that meets all user-specified criteria (e.g., searching for a specific Vaccination Product Name and Administration Date will only return records containing at least one Vaccination entry with matching values in both fields).',
+    fields: [
+      {
+        name: 'vaccine-group',
+        title: 'vaccine group',
+        type: 'select',
+        options: ['COVID-19']
+      },
+      {
+        name: 'product-name',
+        title: 'product name',
+        type: 'select',
+        options: ['Moderna COVID-19 Vaccine', 'Pfizer-BioNTech COVID-19 Vaccine', 'Janssen (J&J) COVID-19 Vaccine', 'Unknown']
```

Comment:
I think we should get these options from `config.vaccine_standards` which is defined in `vaccines.yml`

Reply:
no go on this because that file isn't served to the front end.  this can be done when we move this to the backend (separate ticket)